pkg/aversiveplusplus/Makefile

```diff
@@ -0,0 +1,11 @@
+PKG_NAME=aversiveplusplus
+PKG_URL=https://github.com/AversivePlusPlus/AversivePlusPlus
+PKG_VERSION ?= 86e53477ff98a9e111c7cf612f8d248320dadbd7
```

Comment:
As PKG_VERSION is shared between multiple packages, maybe it is safer to keep this as non-conditional assignment. What do you think?